errand-routes.go

```diff
@@ -10,41 +10,53 @@ import (
 	utils "github.com/polygon-io/errands-server/utils"
 )
 
+const INACTIVE = "inactive"
```

Comment:
Could we not make constants all uppercase. Its not conventional. Casing is only used to signal encapsulation, and since this is the main package we aren't exporting it.

Reply:
Sorry, language confusion and the linter liked it.  I'll fix this.